portal/server/utils/csv-utils.js

```diff
@@ -20,7 +20,12 @@ const columnIndexToExcelColumn = (index) => {
  */
 const extractCellValue = (cell) => {
   const cellValue = cell.value?.result ?? cell.value;
-  const cellValueWithoutNewLines = typeof cellValue === 'string' ? cellValue?.replace(/\r\n|\r|\n/g, ' ').replace(/,/g, '').trim() : cellValue;
+  const cellValueWithoutNewLines = typeof cellValue === 'string'
+    ? cellValue
+      ?.replace(/\r\n|\r|\n/g, ' ')
```

Comment:
I know this was just a prettier change, but I think we can remove the `?` here as the `cellValue` will definitely be defined when the type is `'string'`.